python/foglamp/services/south/server.py

```diff
@@ -178,6 +177,10 @@ def __init__(self):
                                                                   key=data['key'],
                                                                   readings=data['readings']))
                 # pollInterval is expressed in milliseconds
+                if int(self._plugin_handle['pollInterval']['value']) <= 0:
+                    _LOGGER.exception('Plugin {} pollInterval has incorrect value "{}", default choosen as 1000 ms'.
```

Comment:
change to warning

Reply:
_LOGGER.warning('Plugin {} pollInterval must be greater than 0, defaulting to 1000 ms'.